tests/Microsoft.Bot.Schema.Tests/Teams/MessageActionsPayloadTests.cs

```diff
@@ -0,0 +1,388 @@
+    /// </summary>
+    public class MessageActionsPayloadTests
+    {
+        /// <summary>
+        /// Ensures the constructor of the <see cref="MessageActionsPayload"/> class works as expected.
+        /// </summary>
+        [Fact]
+        public void TestMessageActionsPayloadConstructor()
+        {
+            var messageActionPayload = new MessageActionsPayload();
+
+            Assert.NotNull(messageActionPayload);
+        }
+
+        /// <summary>
+        /// Ensures the constructor of the <see cref="MessageActionsPayload"/> class works as expected with arguments.
+        /// </summary>
+        [Fact]
+        public void TestMessageActionsPayloadConstructorWithArguments()
+        {
+            var messageActionPayload = new MessageActionsPayload
+            {
+                Id = "testId",
+                CreatedDateTime = DateTime.Today.ToString(),
+                Deleted = false,
+                Importance = "normal",
+                Locale = "en-us",
+                From = new MessageActionsPayloadFrom(),
+                Body = new MessageActionsPayloadBody(),
+                Attachments = new List<MessageActionsPayloadAttachment>(),
+                Mentions = new List<MessageActionsPayloadMention>(),
+                LinkToMessage = new Uri("https://teams.microsoft.com/l/message/testing-id")
+            };
+
+            Assert.NotNull(messageActionPayload);
+        }
+
+        /// <summary>
+        /// Ensures that the Id property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetId()
+        {
+            var id = "testId";
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "composeExtension/submitAction",
+                ChannelData = new MessagingExtensionAction { MessagePayload = new MessageActionsPayload { Id = id } }
+            };
+
+            var channelData = activity.GetChannelData<MessagingExtensionAction>();
+
+            Assert.Equal(id, channelData.MessagePayload.Id);
+        }
+
+        /// <summary>
+        /// Ensures that the ReplyToId property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetReplyToId()
+        {
+            var replyToId = "replyId";
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "composeExtension/submitAction",
+                ChannelData = new MessagingExtensionAction { MessagePayload = new MessageActionsPayload { ReplyToId = replyToId } }
+            };
+
+            var channelData = activity.GetChannelData<MessagingExtensionAction>();
+
+            Assert.Equal(replyToId, channelData.MessagePayload.ReplyToId);
+        }
+
+        /// <summary>
+        /// Ensures that the MessageType property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetMessageType()
```

Comment:
This test seems to be testing MessagingExtensionAction, the focus of this test class is to test MessageActionPayload, I would suggest we remove it.

Reply:
Done. Thanks!